.github/workflows/vllm_ascend_test_pr_light.yaml

```diff
@@ -108,30 +108,22 @@ jobs:
       - name: Install vllm-project/vllm from source
         working-directory: ./vllm-empty
         run: |
-          VLLM_TARGET_DEVICE=empty python3 -m pip install . --extra-index https://download.pytorch.org/whl/cpu/
+          VLLM_TARGET_DEVICE=empty python3 -m pip install .
           python3 -m pip uninstall -y triton
 
       - name: Checkout vllm-project/vllm-ascend repo
         uses: actions/checkout@v6.0.0
 
       - name: Install vllm-project/vllm-ascend
         run: |
-          export TMPDIR=/mnt
-          rm -f /usr/share/dotnet
-          rm -rf /opt/ghc
-          rm -rf "/usr/local/share/boost"
-          rm -rf "$AGENT_TOOLSDIRECTORY"
-          export PIP_EXTRA_INDEX_URL=https://mirrors.huaweicloud.com/ascend/repos/pypi
-          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/Ascend/ascend-toolkit/latest/x86_64-linux/devlib
-          python3 -m pip install -r requirements-dev.txt --extra-index https://download.pytorch.org/whl/cpu/
-          python3 -m pip install -v . --extra-index https://download.pytorch.org/whl/cpu/
+          python3 -m pip install -r requirements-dev.txt
```

Comment:
`export PIP_EXTRA_INDEX_URL=https://mirrors.huaweicloud.com/ascend/repos/pypi` better to keep this to support torch dev support